specs/ingestion/common/schemas/authentication.yml

```
@@ -106,10 +106,11 @@ AuthenticationType:
   enum: ['googleServiceAccount', 'basic', 'apiKey', 'oauth', 'algolia']
 
 Platform:
-  type: string
-  nullable: true
-  description: Describe which platform the Authentication is used for.
-  enum: ['bigcommerce', 'commercetools']
+  oneOf:
+    - type: string
+      description: Describe which platform the Authentication is used for.
+      enum: ['bigcommerce', 'commercetools']
```

can we force the eslint rule for any length of line?

---

wdym ?

---

I mean so that we always have the same format whether it first in a single line or not

```suggestion
      enum:
        - bigcommerce
        
```

---

The rule doesn't seem to exists https://www.npmjs.com/package/eslint-plugin-yml

---

:( too bad